### PR TITLE
Endpoints overview UI update

### PR DESCRIPTION
**Pull request by sergioc**

### CODE REVIEW

**Comment by sergioc** [comment]
Only need @WojcikMike to make the failed messages status icon link to the respective endpoint's failed messages group.

**Comment by sergioc** [comment]
I think it's best to deal with the failed msgs link in its own issue (not really related to the scope of presentation here)

File: src/ServicePulse.Host/app/js/app.constants.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
         .constant('showPendingRetry', false)
         .constant('scConfig', {
             service_control_url: 'http://localhost:33333/api/',
-            monitoring_urls: ['http://localhost:33633/']
+            monitoring_urls: ['http://localhost:1234/']
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/js/app.constants.js]: Can we undo this change?

Comment (sergioc) [inline src/ServicePulse.Host/app/js/app.constants.js]: We can, but if I undo the monitoring server won't connect to SP, at least for me. Are you sure?

Comment (WojcikMike) [inline src/ServicePulse.Host/app/js/app.constants.js]: now the question is do we want to, as SC.Monitoring while being run from VS use 1234 as a port

Comment (WilliamBZA) [inline src/ServicePulse.Host/app/js/app.constants.js]: SC.Monitoring should change to have it's default be reasonable. 1234 might work for now, but 33633 will be easier for development and debugging later. Once SC.Monitoring has been installed, it is extremely unlikely that 1234 will remain the port, whereas 33633 has a much higher chance.

Comment (WojcikMike) [inline src/ServicePulse.Host/app/js/app.constants.js]: Ok let us change default port of SC.Monitoring to be 33633, and let's revert this change.

Comment (sergioc) [inline src/ServicePulse.Host/app/js/app.constants.js]: Done.

**Comment by sergioc** [comment]
@WilliamBZA @WojcikMike please merge if happy.
